editors/jetbrains/README.md

```diff
@@ -0,0 +1,41 @@
+### Configure External Tool
+
+Go to *File | Settings | Tools | External Tools* for Windows and Linux or *WebStorm | Preferences | Tools | External Tools* for OS X and click **+** to add a new tool. Let’s name it **Prettify**.
+
+#### If you installed prettier globally
+
+In **Program** set `prettier`
+
+In **Parameters** set `--write [other opts] $FilePathRelativeToProjectRoot$` 
+
+In **Working directory** set `$ProjectFileDir$`
+
+![With `prettier`](https://github.com/jlongster/prettier/tree/master/editors/jetbrains/with-prettier.png "With Prettier")
+
+#### If you are using npm script to run prettier
+
+In **Program** set `npm`
+
+In **Parameters** set `run prettify $FilePathRelativeToProjectRoot$` (I'm using `prettify` script for this example)
```

Comment:
We may want to figure out another way to run this. The performance of `npm run` is really bad. I'm not sure what is the best solution here though :x

https://github.com/jlongster/prettier/issues/443